templates/profile-edit.html

```diff
@@ -95,8 +95,8 @@
     </div>
     <script>
         document.addEventListener("DOMContentLoaded",() => {
-            document.getElementById('imagefile').addEventListener('change', (event) => {
-                image = document.getElementById('imagefile')
+            document.getElementById('profile-file').addEventListener('change', (event) => {
+                image = document.getElementById('profile-file')
```

Comment:
Why not "profile-image"?

Reply:
If it is not solved do not resolve it.
If you have objection, suggestion write your comment here.
If you accept the suggestion then change it.
Please do not leave it as it was.

Reply:
because another element is profile-image

![image](https://user-images.githubusercontent.com/112652150/229364835-ca5c1329-792d-471b-a263-756898a67897.png)